sphinx_rtd_theme/layout.html

```diff
@@ -207,6 +207,20 @@
   </script>
   {% endif %}
 
+  {% if theme_analytics_id %}
+  <!-- Theme Analytics -->
+  <script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+  ga('create', '{{ theme_analytics_id }}', 'auto');
+  ga('send', 'pageview');
+  </script>
+
+  {% endif %}
```

Comment:
I think RTD is the proper place for this. Removing it from RTD will then remove it from non RTD theme projects, which we don't want to do. Isn't the official mechanism the "Analytics Code" on advanced project settings?

Perhaps there is a way to do this conditionally -- that is, if running on RTD, the analytics block above doesn't load the script or set up GA. Instead, it either sets the GA user code or the template below is changed to look for this user GA in `conf.py` settings.

https://github.com/rtfd/readthedocs.org/blob/499a74076f0debae09acc1d4c4eeeab03f8f557e/readthedocs/doc_builder/templates/doc_builder/include.js.tmpl#L9